aodntools/timeseries_products/aggregated_timeseries.py

```diff
@@ -422,17 +422,17 @@ def main_aggregator(files_to_agg, var_to_agg, site_code, input_dir='', output_di
 if __name__ == "__main__":
 
     parser = argparse.ArgumentParser(description="Aggregate ONE variable from ALL instruments from ALL deployments from ONE site")
-    parser.add_argument('-site', dest='site_code', help='site code, like NRMMAI',  required=True)
-    parser.add_argument('-var', dest='varname', help='variable to aggregate, like TEMP', required=True)
-    parser.add_argument('-files', dest='filenames',
+    parser.add_argument('--site', dest='site_code', help='site code, like NRMMAI',  required=True)
+    parser.add_argument('--var', dest='varname', help='variable to aggregate, like TEMP', required=True)
+    parser.add_argument('--files', dest='filenames',
                         help='name of the file that contains the source URLs (relative to inpath, if given)',
                         required=True)
-    parser.add_argument('-indir', dest='input_dir', help='base path of input files', default='', required=False)
-    parser.add_argument('-outdir', dest='output_dir', help='path where the result file will be written. Default ./',
+    parser.add_argument('--indir', dest='input_dir', help='base path of input files', default='', required=False)
+    parser.add_argument('--outdir', dest='output_dir', help='path where the result file will be written. Default ./',
```

Comment:
Similarly, for optional arguments. We can also add short options, and `required=False` is [unnecessary as it's the default for an "option" anyway](https://docs.python.org/3.5/library/argparse.html#required))
```suggestion
    parser.add_argument('-i', '--input_dir', help='base path of input files', default='')
    parser.add_argument('-o', '--output_dir', help='path where the result file will be written. Default ./',
                        default='./')
```